model/riscv_sys_regs.sail

```diff
@@ -220,81 +219,80 @@ function get_mstatus_SXL(m : Mstatus) -> arch_xlen = {
   else m.bits[35 .. 34]
```

Comment:
```suggestion
  else m[SXL]
```

I think in future I think we can just use `m[SXL]` directly and initialise it to the RV32 value on RV32: https://github.com/riscv/sail-riscv/pull/597/files#diff-e1a3fe7050c3af4c308a5e39bfb7f0f31094022b9892cee9a611bcb57ccb6726R283